app/presenters/blacklight/document_presenter.rb

```diff
@@ -46,7 +46,7 @@ def heading
 
       fields = Array.wrap(view_config.title_field) + [configuration.document_model.unique_key]
       f = fields.lazy.map { |field| field_config(field) }.detect { |field_config| field_presenter(field_config).any? }
-      field_value(f, except_operations: [Rendering::HelperMethod])
+      f ? field_value(f, except_operations: [Rendering::HelperMethod]) : ""
```

Comment:
Does this need to allocate a string or would a nil value (indicating no header) be a better?

Reply:
@jcoyne , I want it to have same behavior as the old code for `presenter.label` which used to return an empty string in this context.  I haven't tested locally what would happen for the nil case.

I think that in terms of backwards compatibility it `presenter.label` it would thus make sense to continue to return and empty string.

Reply:
Note that I agree that returning `nil` will be a cleaner solution, and originally I did that. But I decided to play it safely and just keep the old behavior.